src/stream/src/executor/source/source_executor.rs

```diff
@@ -179,20 +178,31 @@ impl<S: StateStore> SourceExecutor<S> {
         for sc in &split_change {
             // SplitImpl is identified by its id, target_state always follows offsets in cache
             // here we introduce a hypothesis that every split is polled at least once in one epoch
```

Comment:
please remove the comments, after migrating to state table, we no longer need the hypothesis